lib/scrape.js

```diff
@@ -281,13 +298,33 @@ async function fetchOrgsWithData() {
 
   const fetchingAll = orgs.map(async (org, index) => {
     const existingOrg = existingData.find(existing => existing.id === org.id)
-    const fetchingUsers = orgLeaders[index].map(user => {
+    const fetchingUsers = orgLeaders[index].map(async user => {
       if (existingOrg && existingOrg.leaders) {
         const existingUser = existingOrg.leaders.find(
           leader => leader.id === user.id
         )
-        if (existingUser && existingUser.github_account) {
-          return existingUser.github_account
+        if (
```

Comment:
move this out to a new function so as to not over-complicate this function with GitHub stuff , and also a new function will be testable.